orbit/cmd/orbit/orbit.go

```diff
@@ -516,6 +518,36 @@ func main() {
 		}
 		g.Add(flagRunner.Execute, flagRunner.Interrupt)
 
+		// only setup extensions autoupdate if we have enabled updates
+		if !c.Bool("disable-updates") || c.Bool("dev-mode") {
```

Comment:
Why was this added as a separate `if` clause? (instead of adding it to the existing `if` clause in line 274)

Reply:
The one at like `274` happens much more other stuff like certs, tls proxy, enroll secret, enroll to fleet, etc is configured. And since these extensions are dynamically supplied via agent options, we need orbit to be enrolled first.

Reply:
Gotcha. Let's add that as comment.

Reply:
updated